bescort.lic

```diff
@@ -1447,6 +1447,8 @@ class Bescort
   end
 
   def take_xing_ferry(mode)
+    compendium_type = get_settings.compendium_type
```

Comment:
I don't think we want to get calling `get_settings` this many times. Could you cache it?

Reply:
I tried a couple different methods that didn't work. Do you have suggestions on how to implement?

Reply:
The simplest would be something like

```
settings = get_settings
compendium_type = settings.compendium_type
textbook_type = settings.textbook_type
use_textbook = settings.textbook
```